website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -526,6 +586,58 @@ spec:
           type: 'array<string>',
           description: 'A fixed list of datacenters to try during failover.',
         },
+        {
+          name: 'Targets',
+          type: 'array<ServiceResolverFailoverTarget>',
+          description: `A fixed list of failover targets to try during
+                          failover. It allows operators to express complicated failover
+                          scenarios such as between admin partitions, datacenters and cluster
+                          peers.`,
+          children: [
+            {
+              name: 'Service',
+              type: 'string: ""',
+              description:
+                'The service to use instead of the default of the failover target. If empty the current service is used.',
+            },
+            {
+              name: 'ServiceSubset',
+              type: 'string: ""',
+              description:
+                'The named subset to use for the failover target. If empty the default subset for the requested service is used.',
+            },
+            {
+              name: 'Namespace',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The namespace to use for the failover target. If empty the default namespace is used.',
+            },
+            {
+              name: 'Partition',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The admin partition to use for the failover target. If empty the default partition is used.',
+            },
+            {
+              name: 'Datacenter',
+              type: 'string: ""',
+              description:
+                'The datacenter to use for the failover target. If empty the current datacenter is used.',
+            },
+            {
+              name: 'Peer',
+              type: 'string: ""',
+              description:
+              `Specifies the destination cluster peer to resolve the service
+                  from. When the peer is specified, the failover target relies on Envoy's
+                  outlier detection for health checks. Consul relies on outlier
+                  detection because imported service health checks do not account
+                  for service routers, splitters and resolvers.`,
```

Comment:
My attempt at clarifying:
- What Envoy outlier detection is for someone unfamiliar with the concept
- What an imported service health check is

```suggestion
              `Specifies the destination cluster peer to resolve the target service name from.
                  When the peer name is specified, Consul uses Envoy's outlier detection
                  to determine the health of the failover target based on
                  whether communication attempts to the failover target are generally successful.
                  Service health checks imported from a peer are not considered in the health
                  of a failover target because they do not account for service routers, splitters, and resolvers
                  that may be defined in the peer for the target service.`,
```